packages/core/src/queries/subject-token.ts

```diff
@@ -0,0 +1,14 @@
+import { SubjectTokens } from '@logto/schemas';
+import type { CommonQueryMethods } from '@silverhand/slonik';
+
+import { buildInsertIntoWithPool } from '#src/database/insert-into.js';
+
+export const createSubjectTokenQueries = (pool: CommonQueryMethods) => {
+  const insertSubjectToken = buildInsertIntoWithPool(pool)(SubjectTokens, {
+    returning: true,
+  });
+
+  return {
+    insertSubjectToken,
+  };
+};
```

Comment:
shall we use `class`?

Reply:
Well, `function` looks simpler for me, any ideas? @logto-io/eng